webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -150,166 +153,155 @@ class ManageManifestModal extends Component {
     };
 
     return (
-      <Modal show={this.state.showModal} onHide={this.hideModal}>
-        <Modal.Header>
+      <ForemanModal id={MANAGE_MANIFEST_MODAL_ID} title="Manage Manifest">
```

Comment:
The Model doesn't show the title.
Also the title should be translated

Reply:
@MariaAga Thanks, good catch :)

I updated the Foreman PR to fix the title not showing, and I added translate here.